app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -182,3 +183,48 @@ table
       padding: 4px
       font-size: 0.8em
       color: #505050
+
+// tooltips with html support
+.advanced--tooltip
```

Comment:
I already linked you to the guidelines we are using as a naming scheme (http://www.hagenburger.net/BLOG/Modular-CSS-Class-Names.html).

Looking at this class name that would mean that we have a component called `advanced`, because you are using two dashes. Please use only a single dash if the words belong together: `advanced-tooltip`